src/msw/webview_ie.cpp

```diff
@@ -853,25 +857,190 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
-void wxWebViewIE::RunScript(const wxString& javascript)
+bool wxWebViewIE::MSWSetEmulationLevel(bool defaultLevel)
+{
+    wxRegKey key(wxRegKey::HKCU, _T(REGISTRY_IE_PATH));
```

Comment:
Please don't use `_T()` in the new code at all, we use `wxT()` as it's less confusing now.

In this particular case, `wxT()` should also be part of `REGISTRY_IE_PATH` macro itself.